src/app/components/reminder.js

```diff
@@ -3,36 +3,47 @@ import notify from './../base/notify';
 import { notes } from './../constants/notes';
 import { messages } from './../constants/messages';
 
-export default () => {
-    const APIURl = 'http://localhost:5003/reminders';
-    const day = document.querySelector('[data-reminder="day"]');
-    const time = document.querySelector('[data-reminder="time"]');
-    const setBtn = document.querySelector('[data-reminder="set"]');
-    setBtn.disabled = true;
-    let SELECTEDDAY = 'Select day';
-    let SELECTEDTIME = 'Select time';
-
-    day.addEventListener('change', (event) => {
+export default class Reminder {
+    constructor(container) {
+        this.container = container;
+        this.apiUrl = this.container.getAttribute('data-apiUrl');
+        this.day = this.container.querySelector('[data-reminder="day"]');
+        this.time = this.container.querySelector('[data-reminder="time"]');
+        this.setBtn = this.container.querySelector('[data-reminder="set"]');
+        this.setBtn.disabled = true;
+        this.SELECTEDDAY = 'Select day';
+        this.SELECTEDTIME = 'Select time';
+
+        this.day.addEventListener('change', this.selectDay);
+        this.time.addEventListener('change', this.selectTime);
+        this.setBtn.addEventListener('click', this.setReminder);
+    }
+
+    selectDay = (event) => {
         const selectEl = event.target;
-        SELECTEDDAY = selectEl.value;
-        (SELECTEDDAY !== 'Select day') && (SELECTEDTIME !== 'Select time') ? setBtn.disabled = false : setBtn.disabled = true;
-    });
+        this.SELECTEDDAY = selectEl.value;
 
-    time.addEventListener('change', (event) => {
+        (this.SELECTEDDAY !== 'Select day') && (this.SELECTEDTIME !== 'Select time') ? this.setBtn.disabled = false : this.setBtn.disabled = true;
+    }
+
+    selectTime = (event) => {
         const selectEl = event.target;
-        SELECTEDTIME = selectEl.value;
-        (SELECTEDDAY !== 'Select day') && (SELECTEDTIME !== 'Select time') ? setBtn.disabled = false : setBtn.disabled = true;
-    });
+        this.SELECTEDTIME = selectEl.value;
+
+        (this.SELECTEDDAY !== 'Select day') && (this.SELECTEDTIME !== 'Select time') ? this.setBtn.disabled = false : this.setBtn.disabled = true;
+    }
```

Comment:
`selectDay` and `selectTime` functions do the same thing.  Use just one function.

Also it is not good to compare the `select` option to some string. First you assign a value to `this.SELECTEDDAY` and then in a function you assign it the same value from HTML, which by the way could be different in different languages, so your code would not work if you have it hardcoded in your code. It would be better to get a `value` attribute from the `select` option.

Reply:
And it would be good to not have a `:hover` effect on a `disabled` button